applications/actian-zen/helm-charts/actianzen/values.yaml

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2020 Actian Corporation
+
+replicaCount: 1
+
+image:
+  name: actianzen
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
```

Comment:
Comment is unclear. Is the user need to take care of that? Can you clarify how the user can identify the correct tag?

Reply:
Comment removed as it was confusing. Currently we are providing archives of the containers with only one tag so elaborating on this isn't necessary at the moment.